templates/container/container.py

```diff
@@ -60,12 +61,16 @@ def install(self):
         for env in self.data['env']:
             envs[env['name']] = env['value']
 
-        self._node_sal.containers.create(self.name, self.data['flist'], hostname=self.data['hostname'],
-                                        mounts=mounts, nics=self.data['nics'],
+        self._container = self._node_sal.containers.create(self.name, self.data['flist'], hostname=self.data['hostname'],
+                                        mounts=mounts, nics=copy.deepcopy(self.data['nics']),
                                         host_network=self.data['hostNetworking'],
                                         ports=ports, storage=self.data['storage'],
                                         init_processes=self.data['initProcesses'],
-                                        privileged=self.data['privileged'], env=envs)
+                                        privileged=self.data['privileged'], identity=self.data['ztIdentity'],
+                                        env=envs)
+        self.data['ztIdentity'] = self._container_sal.identity
+        self._container_sal.authorize_networks(self.data['nics'])
```

Comment:
We should not try authorize the container to the zerotier network directly from here.
This will need to be done by the 3bot that ask for the container. For more detail see https://github.com/threefoldtech/0-templates/issues/201#issuecomment-444939008

Reply:
I removed the authorization part from the template. The rest looks good?